wizard/src/utils/heuristics.ts

```diff
@@ -14,13 +14,89 @@
  * limitations under the License.
  */
 
-import { CsvData, MappedThing, Mapping } from "../types";
+import {
+  Column,
+  ConfidenceLevel,
+  CsvData,
+  DCProperty,
+  DCType,
+  DetectedDetails,
+  MappedThing,
+  Mapping,
+  MappingType,
+  MappingVal,
+} from "../types";
+import { PlaceDetector } from "./detect_place";
 
 /**
- * Given a csv, returns the predicted mapping
+ * Process all columns and return the one which best represents the detected
+ * Place along with its details. If no Place is detected, the return value is
+ * null.
  *
- * @param csv array of csv rows where each row is an array of strings
+ * @param columns is a mapping from column indices to (sampled) column values.
```

Comment:
nit: either change this to "cols" or change the param to "columns"

Reply:
done